v-spec.adoc

```diff
@@ -4535,7 +4535,7 @@ if _OFFSET_ < `vl`.
 
    OFFSET is amount to slideup, either from x register or a 5-bit immediate
 
-                    0 <  i < max(vstart, OFFSET)  Unchanged
+                    0 <=  i < max(vstart, OFFSET)  Unchanged
   max(vstart, OFFSET) <= i < vl                   vd[i] = vs2[i-OFFSET] if v0.mask[i] enabled
                    vl <= i < VLMAX                Follow tail policy
```

Comment:
There is a general mandate that tail elements are undisturbed when `vstart >= vl`. I think we should mention "assuming `vstart < vl`"  in the heading for this description.